node_modules/oae-principals/lib/test/util.js

```diff
@@ -270,6 +270,18 @@ var assertGetUserFails = module.exports.assertGetUserFails = function(restCtx, u
 var assertGetMeSucceeds = module.exports.assertGetMeSucceeds = function(restCtx, callback) {
     RestAPI.User.getMe(restCtx, function(err, me, response) {
         assert.ok(!err);
+
+        // Sanity check the me model
+        assert.ok(me);
+        assert.ok(me.tenant);
+        assert.ok(_.isString(me.tenant.alias));
+        assert.ok(_.isString(me.tenant.displayName));
+        if (me.tenant.alias === 'guest') {
+            assert.strictEqual(me.tenant.isGuestTenant, true);
+        } else {
+            assert.ok(!_.chain(me.tenant).keys().contains('isGuestTenant').value());
```

Comment:
Can't we just do `assert.ok(!me.tenant.isGuestTenant)` / `assert.ok(_.isUndefined(me.tenant.isGuestTenant))` ?